x-pack/plugins/observability_solution/apm/server/utils/get_agent_name.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export function getAgentName(
```

Comment:
We have a package for agent utils, https://github.com/elastic/kibana/blob/main/packages/kbn-elastic-agent-utils/README.md. What do you think of adding this util in the package?

Reply:
good catch, definitely missed that one, will do!

Reply:
done!